core/src/main/java/io/questdb/log/Log.java

```diff
@@ -28,22 +28,42 @@ public interface Log {
 
     LogRecord advisory();
 
+    default void advisoryUtf8(long lo, long hi) {
```

Comment:
it would be nice to put @SuppressWarnings("unused") and comment that it is being used from the jni side.